.github/actions/spelling/excludes.txt

```diff
@@ -54,14 +54,26 @@ SUMS$
 \.zip$
 ^consolegit2gitfilters\.json$
 ^dep/
-^oss/
+^doc/reference/master-sequence-list.csv$
 ^doc/reference/UTF8-torture-test\.txt$
+^oss/
+^src/host/ft_uia/run\.bat$
+^src/host/runft\.bat$
+^src/host/runut\.bat$
 ^src/interactivity/onecore/BgfxEngine\.
 ^src/renderer/wddmcon/WddmConRenderer\.
+^src/terminal/adapter/ut_adapter/run\.bat$
+^src/terminal/parser/delfuzzpayload\.bat$
+^src/terminal/parser/ft_fuzzer/run\.bat$
 ^src/terminal/parser/ft_fuzzer/VTCommandFuzzer\.cpp$
-^src/types/ut_types/UtilsTests.cpp$
```

Comment:
@zadjii-msft: for reference, this file was already being ignored (I spent some time trying to find the advice that automatically ignored it and couldn't ... because it wasn't automatic ... sigh)